domain-get/index.ts

```diff
@@ -1,12 +1,40 @@
 import { AzureFunction, Context, HttpRequest } from "@azure/functions";
-import { doServiceOperation, getDomain } from "../src/services/domainService";
-
+import { DomainId } from "@zero-tech/data-store-core";
+import { getMongoDomainService } from "../src/helpers";
+import { getDomainFindOptionsFromQuery } from "../src/helpers/domainFindOptionsHelper";
+import { validateDomainId } from "../src/schemas";
 const httpTrigger: AzureFunction = async function (
   context: Context,
   req: HttpRequest
 ): Promise<void> {
   context.log("HTTP trigger function processed a request.");
-  await doServiceOperation(getDomain, context, req);
+  try {
+    const domainId: DomainId = req.params.id;
+    if (!validateDomainId(domainId)) {
```

Comment:
These lines `12-19` seem repeated a lot, could we refactor into a helper?

Reply:
Added a helper, though only 2 lines were able included